modules/core/src/lib/deck.ts

```diff
@@ -1007,9 +1007,14 @@ export default class Deck<ViewsT extends ViewOrViews = null> {
 
     this.deckPicker = new DeckPicker(this.device);
 
+    const widgetContainer = document.createElement('div');
+    widgetContainer.id = 'deck-widgets-container';
+    const parent = this.props.parent?.parentElement || document.body;
```

Comment:
I don't think we want to do this. How would we attach the widget container otherwise?